docs/installation/introduction.rst

```diff
@@ -1,6 +1,104 @@
 Introduction
 ============
 
+Foreword
+^^^^^^^^
+MetalK8s is a Kubernetes_ distribution with a number of addons carefully
+picked for optimal on-premises deployments, including pre-configured monitoring
+and alerting, self-healing system configuration, and more.
+
+The installation of a MetalK8s cluster can be broken down into
+the following steps:
+
+#. :doc:`Setup <./setup>` of the environment (with requirements and example
+   OpenStack deployment)
+#. :doc:`Deployment <./bootstrap>` of the :term:`Bootstrap node`, the first
+   machine in the cluster
+#. :doc:`Expansion <./expansion>` of the cluster, orchestrated from the
+   Bootstrap node
+
+.. _Kubernetes: https://kubernetes.io/
+
+.. _installation-intro-architecture:
+
+Choosing a Deployment Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before starting the installation, choosing an architecture is recommended, as
+it can impact sizing of the machines and other infrastructure-related details.
+
+.. note:: "Machines" may indicate bare-metal servers or VMs interchangeably
+
+Standard Architecture
+"""""""""""""""""""""
+MetalK8s is designed with on-premise, offline, reliable deployments in mind.
+The first example architecture proposed follows this spirit, focusing on
+reliability rather than compacity:
+
+- One machine dedicated to running Bootstrap services (see
+  :ref:`the Bootstrap role<node-role-bootstrap>` definition below)
+- Three extra machines (or five if installing a really large cluster,
+  100+ nodes) for running the Kubernetes_ control plane (with
+  :ref:`core K8s services<node-role-master>` and the backing
+  :ref:`etcd DB<node-role-etcd>`)
+- One or more machines dedicated to running Infra services (see
```

Comment:
Shouldn't we recommend to have at least 2 or 3 nodes for infra services? to have a minimum of resilience in case of HW failure?

Reply:
We can, but for now, this requires manual changes that won't be persisted on upgrade.

Reply:
Okay understood